forge-gui/src/main/java/forge/player/HumanPlaySpellAbility.java

```diff
@@ -109,7 +109,9 @@ public final boolean playAbility(final boolean mayChooseTargets, final boolean i
             ability.setPaidLife(0);
         }
 
-        ability = GameActionUtil.addExtraKeywordCost(ability);
+        if (ability.isSpell() && !c.isCopiedSpell()) {
```

Comment:
this is redundant as addExtraKeywordCost already checks it at the start

Reply:
Yeah, just to be sure in case I might touch that in the future